### PR TITLE
bpo-40901: Describe what "interface name" means on Windows

### PR DESCRIPTION
**Pull request by jstasiak**
https://bugs.python.org/issue40901

### CODE REVIEW

**Comment by jstasiak** [comment]
cc @ZackerySpytz @zooba (as you were involved in https://github.com/python/cpython/pull/13522) I'd appreciate some guidance here. I think news entry can be skipped safely in this case.

**Comment by jstasiak** [comment]
Should I ask someone else for a review? Not sure who, I'm thinking people who changed this module's documentation recently.

File: Doc/library/socket.rst [hunk 1]
```diff
@@ -1091,6 +1091,17 @@ The :mod:`socket` module also offers various network-related services:
...
+   .. note::
+
+      On Windows network interfaces have different names in different contexts:
+
+      * UUID: ``{FB605B73-AAC2-49A6-9A2F-25416AEA0573}``
+      * name: ``ethernet_32770``
+      * friendly name: ``vEthernet (nat)``
+      * description: ``Hyper-V Virtual Ethernet Adapter``
+
+      This function returns names of the second form from the list.
```
Comment (taleinat) [inline Doc/library/socket.rst]: This last sentence is perhaps not as clear as it could be. Perhaps mention the example value `ethernet_32770` explicitly?

**Comment by taleinat** [comment]
> I think news entry can be skipped safely in this case.

Indeed.

File: Doc/library/socket.rst (hunk 1 above)
Comment (jstasiak) [reply Doc/library/socket.rst]: That's fair, I'll do that in a minute.

**Comment by zooba** [comment]
Thanks! Very helpful contribution :)

**Pull request merged**